### PR TITLE
Finalize CI Secretes

### PR DESCRIPTION
**Pull request by jdpigeon**
Looks like you did most of the work to put our keys in CI last September, @teonbrooks!

I think the missing piece is that the env variables actually used in the app didn't match what we added to CI. I've updated them here to match, which should be all we need.

### CODE REVIEW

File: configs/webpack.config.main.prod.babel.js [hunk 1]
```diff
@@ -55,13 +55,23 @@ export default merge(baseConfig, {
...
       NODE_ENV: 'production',
       DEBUG_PROD: false,
       START_MINIMIZED: false,
       E2E_BUILD: false,
     }),
+    /**
+     * The keys for emotiv SDK usage are piped into prod builds here via
+     * env variables set by secrets in Github Actions
+     */
+    new webpack.DefinePlugin({
```
Comment (teonbrooks) [inline configs/webpack.config.main.prod.babel.js:70]: ooohh, i was wondering about this step. this makes sense

**Comment by teonbrooks** [comment]
just curious, does the GitHub secrets support json to keep the secrets in one file or is it best specified individually like I did. I had done it both ways but I didn't get it to work (needed to add the part to webpack)

**Comment by teonbrooks** [comment]
closes #141

**Comment by jdpigeon** [comment]
Based on the github docs and another tutorial I saw, storing secrets as strings is the most common way to do things. I don't think storing them as a single json is necessarily desirable.

Another thing I noticed is that we don't actually use the username and password anymore since the update to the new Cortex SDK. We can remove those secrets whenever.

**Pull request merged**